R/centrality.R

```diff
@@ -2121,7 +2121,7 @@ alpha.centrality.sparse <- function(
   )
 
   ## exo
-  exo <- cbind(rep(exo, length.out = vc))
+  exo <- cbind(vctrs::vec_recycle(exo, vc))
```

Comment:
is this covered by a test? curious about this function actually works on a matrix.